pom.xml

```diff
@@ -400,6 +400,17 @@
                 <groupId>org.apache.hadoop</groupId>
                 <artifactId>hadoop-aws</artifactId>
                 <version>${hadoop.version}</version>
+                <!-- Exclude aws bundles, we have declared the dependencies of AWS separately. -->
+                <exclusions>
+                    <exclusion>
+                        <groupId>software.amazon.awssdk</groupId>
+                        <artifactId>bundle</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.amazonaws</groupId>
+                        <artifactId>aws-java-sdk-bundle</artifactId>
```

Comment:
it seems doesn't contains `aws-java-sdk-bundle` module

![image](https://github.com/apache/amoro/assets/95013770/a37c38c4-7dca-4a11-b243-95beb3cee818)